report_qweb_element_page_visibility/views/layouts.xml

```diff
@@ -0,0 +1,57 @@
+<odoo>
+    <template id="minimal_layout" inherit_id="web.minimal_layout">
+        <xpath expr="//t[@t-if='subst_needed']" position="replace">
+            <t t-if="subst is True">
+                <script>
+                    function subst() {
+                        var vars = {};
+                        var x = document.location.search.substring(1).split('&amp;');
+                        for (var i in x) {
+                            var z = x[i].split('=', 2);
+                            vars[z[0]] = unescape(z[1]);
+                        }
+                        var fromPage = document.getElementsByClassName('page');
```

Comment:
in 14.0 this block was added in the core:
```
                        var x = ['sitepage', 'sitepages', 'section', 'subsection', 'subsubsection'];
                        var z = {'sitepage': 'page', 'sitepages': 'topage'};
                        for (var i in x) {
                            var y = document.getElementsByClassName(z[x[i]] || x[i])
                            for (var j=0; j&lt;y.length; ++j)
                                y[j].textContent = vars[x[i]];
                        }
```
https://github.com/odoo/odoo/blob/14.0/addons/web/views/report_templates.xml#L286-L293

Reply:
never mind, was already added in 13.0 and earlier, seems to have no effect at all